examples/pylab_examples/date_index_formatter.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, dates, fmt='%Y-%m-%d'):
 
     def __call__(self, x, pos=0):
         'Return the label for time x at position pos'
-        ind = int(round(x))
```

Comment:
Do you mind if I push to your branch to change the import of numpy to follow the standards?